metrics/lib/features/common/presentation/metrics_theme/widgets/metrics_app.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:flutter/material.dart';
+import 'package:metrics/features/common/presentation/metrics_theme/model/metrics_theme_data.dart';
+import 'package:metrics/features/common/presentation/metrics_theme/widgets/metrics_theme.dart';
+
+enum MetricsThemeType { light, dark }
+
+/// The widget to manage the current [MetricsThemeData].
+class MetricsApp extends StatelessWidget {
```

Comment:
Suggestion: let's start a discussion about the functionality of this Widget and possibility of merging it with MetricsThemeBuilder